<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Publicacao_ui</class>
 <widget class="QWidget" name="Publicacao_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="nomeUsuario">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nome do usu√°rio</string>
   </property>
  </widget>
  <widget class="QLabel" name="dataP">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>dd/MM/yyyy hh:mm</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="descP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>511</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>511</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="imgsPubli"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
